src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -74,11 +81,19 @@ export class HealthComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.interval = this.refreshIntervalService.intervalData$.subscribe(() => {
       this.getHealth();
+      if (!this.ratiosFetched) {
+        this.ratiosFetched = true;
```

Comment:
I don't see the value of `this.ratiosFetched`. If the intention is to call `this.osdService.getOsdDump()` once, then you can put that call outside of the refresh interval subscription.